wicketd/src/rss_config.rs

```diff
@@ -498,7 +503,7 @@ fn validate_rack_network_config(
     // TODO Add more client side checks on `rack_network_config` contents?
 
     Ok(bootstrap_agent_client::types::RackNetworkConfigV1 {
-        rack_subnet: RACK_SUBNET.into(),
+        rack_subnet: RACK_SUBNET.net(),
```

Comment:
Thanks for catching this.

The original code makes me really uncomfortable to be honest, I don't like the fact that there's a From impl here. (Feels like the opposite of misuse-resistant.) Is that something we can fix?